.github/workflows/issue-notifications.yml

```diff
@@ -0,0 +1,51 @@
+name: Issue Notifications
+
+on:
+  workflow_dispatch: # Manual trigger for testing
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: "0 * * * *" # Every hour
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Issue Notification Action
+        uses: joshdholtz/github-action-issue-ack@v15
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          slack_webhook_url: ${{ secrets.ACK_SLACK_WEBHOOK_URL }}
+
+          # Keywords to raise alert
+          title_keywords: ${{ secrets.ACK_ALERT_KEYWORDS }}
+
+          # Excluded labels to silence notifications
+          excluded_labels: "acknowledged"
+
+          # Reaction and comment thresholds to raise alert
+          reaction_threshold: ${{ secrets.ACK_REACTION_THRESHOLD }}
+          comment_threshold: ${{ secrets.ACK_COMMENT_THRESHOLD }}
+
+          # Batch checking configuration
+          check_all_open_issues: "true"
+          max_issues_to_check: "100"
+          issue_state: "open"
+
+          # Custom message for testing
```

Comment:
the "for testing" is confusing me

Reply:
Ha! I copied this from my test project 😛 I will fix